projects/core/src/lib/mask.ts

```diff
@@ -127,6 +131,7 @@ export class Maskito extends MaskHistory {
 
     destroy(): void {
         this.eventListener.destroy();
+        this.teardowns.forEach(teardown => teardown?.());
```

Comment:
Maybe add a try catch and throw an `AggregateError` if an error occurs?

Reply:
I'm not sure it would be beneficial 🤔